app/views/proposals/query.html.haml

```diff
@@ -2,5 +2,10 @@
   = render partial: 'search_ui'
   %h1
     Proposals participated in
+    %h2
+    %strong Currently displaying results for: 
+    = property_to_s(@text)
+    %br
+    %a{ :href => "/proposals" } Clear search terms
```

Comment:
Your `h2` doesn't have a body, so it can be removed. Also, maybe replace the anchor with
```
%a{href: proposals_path} Clear search terms
```
so as to not hard-code the urls.